pkg/builder.go

```diff
@@ -28,7 +27,7 @@ const (
 // Git.
 //
 // Run returns on of the Status* status code constants.
-func Run(sshHostIP string, sshHostPort int, cmd string) int {
+func RunBuilder(sshHostIP string, sshHostPort int, sshServerCircuit *sshd.Circuit) int {
```

Comment:
Also, the reason I had to touch this function in the first place was to add the `sshServerCircuit` parameter. That's the part I missed in my last comment, and that's what related to health checks.